plugins/ProfessionalServices/stylesheets/promos.less

```
@@ -67,4 +67,14 @@
       text-decoration: none;
     }
   }
+
+  .promo-dismiss {
+    padding-top: 16px;
+    .font-default(12px; 21px);
+    color: #686868;
```

```suggestion
    color: @color-silver-l40;
```

That would be `#666666`, but also seems not to be used anywhere else.